app/views/shared/_order_detail_action_form.html.haml

```diff
@@ -1,6 +1,8 @@
 - @facility_users ||= User.find_users_by_facility(current_facility) || []
 
 %td.action-form
+  = render "projects/shared/select_facility_project", facility: current_facility
+  - # TODO: Check if un-used and remove.  This was used by the old projects engine before it was brought into the application.
```

Comment:
`"projects/shared/select_facility_project"` is definitely still used, but since we are rendering it explicitly now I'm not sure if anyone else is using the hook for `"batch_update_above_product_column"` below on line 6